deploy.yml

```diff
@@ -7,6 +7,11 @@
       msg: "Running Ansible"
 
   - name: Install Apache
+    become: true
     apt: 
       name: apache2
-      state: present
+  - name: Add PHP 7.0 repo
```

Comment:
Adding a blank line between the different named blocks would be cool so you can tell where one chunk stops and another starts